guidelines/index.html

```diff
@@ -1432,30 +1432,30 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
  </section>
 
             
-            <section class="guideline" id="focus-appearance"><div class="header-wrapper"><h4 id="x2-4-13-focus-appearance"><bdi class="secno">Success Criterion 2.4.13 </bdi>Focus Appearance</h4><a class="self-link" href="#focus-appearance" aria-label="Permalink for Section 2.4.13"></a></div>
+            <section class="guideline" id="focus-appearance"><div class="header-wrapper"><h4 id="x2-4-13-focus-appearance"><bdi class="secno">達成基準 2.4.13 </bdi>フォーカスの外観</h4><a class="self-link" href="#focus-appearance" aria-label="Permalink for Section 2.4.13"></a></div>
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html">Understanding Focus Appearance</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#focus-appearance">How to Meet Focus Appearance</a></div><p class="conformance-level">(レベル AAA)</p>
 	<p class="change">[New]</p>
     
-    <p>When the keyboard <a href="#dfn-focus-indicator" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-focus-indicator-1" title="[New]">focus indicator</a> is visible, an area of the focus indicator meets all the following:</p>
+    <p>キーボードの<a href="#dfn-focus-indicator" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-focus-indicator-1" title="[New]">フォーカスインジケータ</a>が視覚的に表示される場合、フォーカスインジケータの領域が以下のすべてを満たす。</p>
     <ul>
-        <li>is at least as large as the area of a 2 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-3" title="約 0.0213 度の視野角。">CSS pixel</a> thick <a href="#dfn-perimeter" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-perimeter-1" title="[New]">perimeter</a> of the unfocused component or sub-component, and</li>
-        <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.</li>
+        <li>フォーカスされていないコンポーネント又はサブコンポーネントの厚さ 2 <a href="#dfn-css-pixels" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-css-pixels-3" title="約 0.0213 度の視野角。">CSS ピクセル</a>の<a href="#dfn-perimeter" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-perimeter-1" title="[New]">外周</a>の面積と少なくとも同じ大きさである。かつ、</li>
+        <li>同一ピクセルのフォーカス状態と非フォーカス状態との間で少なくとも 3:1 のコントラスト比がある。</li>
     </ul>
 
-    <p>Exceptions:</p>
+    <p>例外:</p>
     <ul>
-      <li>The focus indicator is determined by the <a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-1" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">user agent</a> and cannot be adjusted by the author, or</li>
-        <li>The focus indicator and the indicator's background color are not modified by the author.</li>
+      <li>フォーカスインジケータが<a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-1" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">ユーザエージェント</a>によって定められており、かつコンテンツ制作者による調整ができない場合。又は、</li>
```

Comment:
determineは素直に「決定する」でよいかと思います。
直後のby the authorを「コンテンツ制作者が」としているので合わせるのがよいかと。

```suggestion
      <li>フォーカスインジケータが<a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-1" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">ユーザエージェント</a>によって決定され、かつコンテンツ制作者が調整できない場合。又は、</li>
```

Reply:
原文が「determined by the user agent」となっている箇所を、ひととおり拾ってみました。WCAG 2.1 の時点で2箇所あって (達成基準 1.4.11 と 2.5.5)、「提示され」「定められ」で揺れています。
個人的には、1.4.11 の「提示されて」は違和感があるかなと思い (「提示」は「presentation」の訳語として、用語集にも定義されている)、であれば「定められ」に統一かな、という気がします。 @bakera さんの案で言うならば、「規定され」かな、という気がします。

- 達成基準 1.4.11 非テキストのコントラスト ... 「ユーザエージェントによって**提示され**ていて」
- 達成基準 2.4.13 Focus Appearance
- 達成基準 2.5.5 Target Size (Enhanced) ...「ユーザエージェントによって**定められ**ており」
- 達成基準 2.5.7 Dragging Movements
- 達成基準 2.5.8 Target Size (Minimum)